dmd-frontend/src/components/Editor.tsx

```diff
@@ -18,7 +21,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     padding: '10px',
     fontFamily: 'Consolas,Courier,serif',
     lineHeight: '21px;',
-    boxSizing: 'border-box',
+    boxSizing: 'border-box'
+  },
+  grid: {
```

Comment:
Please consider it could be an issue, please check to make sure it's mobile responsible.

Reply:
> This task too big, please broke down. Please, use this format for merge request or commit name
> 
> #[Task number] [Title]

Ok, thank you

Reply:
Ok, thank you for suggestions

Reply:
@trinhlaosu I didn't see any changes?

Reply:
I checked and  it's mobile responsible